### PR TITLE
withfield/aconst_init/vnew vm removal

### PR DESCRIPTION
**Pull request by theresa-m**
Depends on https://github.com/eclipse-openj9/openj9/pull/19770

Related to https://github.com/eclipse-openj9/openj9/issues/18316

### CODE REVIEW

File: runtime/verutil/chverify.c [hunk 1]
```diff
@@ -139,25 +134,25 @@ isInitOrClinitOrNewImpl (J9CfrConstantPoolInfo * info)
 /**
  * Determine if this name is "<init>", "<clinit>", or "<vnew>".
  *
- * @returns 0 if name is a normal name, CFR_METHOD_NAME_INIT if '<init>', CFR_METHOD_NAME_CLINIT if '<clinit>', CFR_METHOD_NAME_NEW if '<vnew>', and -1 if it starts with '<' but is not a valid class name.
+ * @returns 0 if name is a normal name, CFR_METHOD_NAME_INIT if '<init>', CFR_METHOD_NAME_CLINIT if '<clinit>', and -1 if it starts with '<' but is not a valid class name.
  * @note result is positive if the name is "<init>", "<clinit>", or "<vnew>", result is negative if the name is illegal
```
Comment (hangshao0) [inline runtime/verutil/chverify.c:138]: >  `or "<vnew>", result is negative if the name is illegal`

`"<vnew>"` should be removed from this line.

Comment (theresa-m) [reply runtime/verutil/chverify.c]: I corrected this and rebased to include the updated tests from https://github.com/eclipse-openj9/openj9/pull/19770

**Comment by hangshao0** [comment]
Jenkins test sanity,extended zlinuxval jdknext

**Comment by hangshao0** [comment]
Jenkins test sanity,extended amac jdk21

**Comment by theresa-m** [comment]
The extended failure is due to https://github.com/eclipse-openj9/openj9/issues/19765

**Review by hangshao0 — APPROVED** [review]
LGTM.

**Pull request merged**